help/destinations/catalog/email-marketing/sendgrid.md

```diff
@@ -0,0 +1,250 @@
+
+* Within the AEP console, navigate to *Destinations*.
+
+* Click the *Catalog* tab and search for *Sendgrid*. Then click *Activate Segments*.
+
+![](../../assets/catalog/email-marketing/sendgrid/02-catalog.jpg)
+
+* You will be shown a wizard where the first screen is to choose a destination. For setting up this connector, let us create a new destination by clicking on *Configure new destination*.
+
+![](../../assets/catalog/email-marketing/sendgrid/03.jpg)
+
+* Select the *New Account* option and fill in the *Bearer Token* value which is the Sendgrid *API Key* value we had previously copied within [prerequisities].(#prerequisites-prerequisites)
+
+![](../../assets/catalog/email-marketing/sendgrid/04.jpg)
+
+* Upon clicking *Connect to destination*, you will be shown the next [step](#connection-parameters-parameters) to fill in additional information fields.
+
+### Connection parameters {#parameters}
+
+While [setting up](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) this destination, you must provide the following information:
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in future.
+*  **[!UICONTROL Description]**: An optional description that will help you identify this destination in future.
+
+![](../../assets/catalog/email-marketing/sendgrid/05.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/06.jpg)
+
+## Connect to destination {#connect}
+
+* To connect to this destination, the generic steps described in the [destination configuration tutorial](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) will provide guidance. See below for the screengrabs corresponding to this destination. 
+
+![](../../assets/catalog/email-marketing/sendgrid/07.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/08.png)
+
+* You will be navigated to the list of destinations.
+
+![](../../assets/catalog/email-marketing/sendgrid/09.jpg)
+
+* Select the newly created destination and click *Next*.
+
+![](../../assets/catalog/email-marketing/sendgrid/10.jpg)
+
+* Select the *Segment* *(created earlier in [prerequisites](#prerequisites-prerequisites))* and click *Next*.
+
+![](../../assets/catalog/email-marketing/sendgrid/11.jpg)
+
+* Now map the *Source* and *Schema* *(created earlier in [prerequisites](#prerequisites-prerequisites))* fields.
+
+![](../../assets/catalog/email-marketing/sendgrid/12.jpg)
+
+> [!IMPORTANT]
+> The *Source* field attribute *"Email"* should be mapped to the *Target* field attribute "external_id" as shown in the subsequent steps.
+![](../../assets/catalog/email-marketing/sendgrid/13.png)
```

Comment:
can we please add some descriptive text between screenshots 13-22 for users to understand what is going on? 

for example, please call out when you are mapping attributes and when you are mapping identitie
also, an important question for screenshots 20-22 - are those all the mappings that are accepted by Sendgrid? if yes, let's please call that out.